panel/util.py

```diff
@@ -37,6 +37,9 @@
 bokeh_version = Version(bokeh.__version__)
 
 
+PARAM_NAME_PATTERN = re.compile('^.*\d{5}$')
```

Comment:
This seems like something that should be internal to Param (a method or function to determine if a name is non-default), not hardcoded into Panel.